lib/models/pad_button_item.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+import 'gestures.dart';
+
+/**
+ * Model of one padd button.
+ */
+class PadButtonItem {
+  /// [index] the key to recognize button instance.
+  final int index;
+
+  /**
+   *   [ buttonText] optional parameter, the text to be displayed inside the
+   *   button. Omitted if [buttonImage] is set. Default value is empty string.
+   */
+  final String buttonText;
+
+  /**
+   *   [buttonImage] optional parameter, image which will be displayed inside
+   *   the button.
+   */
+  final Image buttonImage;
+
+  ///[backgroundColor] color of button in default state.
+  final Color backgroundColor;
+
+  /// [pressedColor] color of button when it is pressed.
+  final Color pressedColor;
+
+  /**
+   *  [supportedGestures] list of gestures for button which will call the callback [PadButtonsView.padButtonPressedCallback]
+   */
+  final List<Gestures> supportedGestures;
+
+  const PadButtonItem(
+      {this.index,
```

Comment:
Maybe we could use @required + assert to indicate what parameters are required to work?